cpp/src/aztec3/circuits/abis/private_kernel/previous_kernel_data.hpp

```diff
@@ -14,6 +14,7 @@ using aztec3::utils::types::NativeTypes;
 using plonk::stdlib::witness_t;
 using std::is_same;
 
+// @todo Naming should not be previous. Annoying.
```

Comment:
This was first named in the context of using this in the Private Kernel Circuit (where "Previous" is a useful disambiguation).
Maybe we should create a type alias `KernelData` for it, so the Rollup circuits can use that name.